pkg/global/apply_configuration.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/client_golang/prometheus/push"
+	"google.golang.org/grpc/codes"
```

Comment:
Nit: Add an empty line above this. The current convention in the Buildbarn codebase is to group imports by domain. github.com != golang.org.

Reply:
Will fix.